docker-compose.yaml

```diff
@@ -6,11 +6,8 @@ services:
         - TENZIR_VERSION=main
     environment:
       - TENZIR_ENDPOINT=tenzir-node:5158
-      - TENZIR_PLUGINS__PLATFORM__API_KEY=${TENZIR_PLUGINS__PLATFORM__API_KEY:-}
-      - TENZIR_PLUGINS__PLATFORM__CONTROL_ENDPOINT=${TENZIR_PLUGINS__PLATFORM__CONTROL_ENDPOINT:-wss://ws.tenzir.app/production}
-      - TENZIR_PLUGINS__PLATFORM__TENANT_ID=${TENZIR_PLUGINS__PLATFORM__TENANT_ID:-}
-    ports:
-      - 5158:5158
+      - TENZIR_TOKEN=${TENZIR_TOKEN:-}
+      - TENZIR_PLATFORM_CONTROL_ENDPOINT=${TENZIR_PLATFORM_CONTROL_ENDPOINT:-wss://ws.tenzir.app/production}
```

Comment:
Does it make sense to duplicate the string here instead of relying on the built-in default of the platform plugin? Then we have to change it twice if we ever want to change the url.

Reply:
Done in https://github.com/tenzir/example-plugin/compare/8d2f47b67ff89da489d28d02515e283adc124ce9..6aa9d7336d25ebd8e201129ca1989fe8b3c172bc